packages/alea-frontend/pages/api/access-control/update-acl.ts

```diff
@@ -19,24 +20,22 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
   if (!(await validateMemberAndAclIds(res, memberUserIds, memberACLIds)))
     return res.status(422).send('Invalid items');
 
-  const numMembershipRows = memberUserIds.length + memberACLIds.length;
-  const values = new Array(numMembershipRows).fill('(?, ?, ?)');
-
-  const memberQueryParams = [];
-  for (const userId of memberUserIds) memberQueryParams.push(id, null, userId);
-  for (const aclId of memberACLIds) memberQueryParams.push(id, aclId, null);
-
-  const memberQuery = `INSERT INTO ACLMembership (parentACLId, memberACLId, memberUserId) VALUES 
-  ${values.join(', ')}`;
   await executeTxnAndEndSet500OnError(
```

Comment:
if the result of this function is falsey/undefined this means that the transaction has failed and that we should do an early return. Please fix here and also in the create API, if needed